src/components/RichTextEditor/serializer/markdown/types.ts

```diff
@@ -130,8 +130,14 @@ export type MarkdownAstNode = {
     checked?: any;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     indent?: any;
+    target?: TargetType;
 };
 
+export enum TargetType {
+    'BLANK' = '_blank',
+    'SELF' = '_self',
+}
```

Comment:
why an enum and not type `'_blank' | '_self`?

Reply:
They are use through the codebase TargetType.BLANK

Reply:
Isn't the enum for fixed constant values?